src/linear_algebra/basic.lean

```diff
@@ -2866,6 +2890,29 @@ def automorphism_group.to_linear_map_monoid_hom :
   map_one' := rfl,
   map_mul' := λ _ _, rfl }
 
+/-- The tautological action by `M ≃ₗ[R] M` on `M`.
+
+This generalizes `function.End.apply_mul_action`. -/
+instance apply_distrib_mul_action : distrib_mul_action (M ≃ₗ[R] M) M :=
+{ smul := ($),
+  smul_zero := linear_equiv.map_zero,
+  smul_add := linear_equiv.map_add,
+  one_smul := λ _, rfl,
+  mul_smul := λ _ _ _, rfl }
+
+instance apply_smul_comm_class : smul_comm_class R (M ≃ₗ[R] M) M :=
```

Comment:
Nitpick: the order of these declarations changes between the various `mul_action` defs, so it's harder to see the analogies between the various groups of instances.

Reply:
Fixed I think; mind double checking, since I did in on mobile?

Reply:
The order looks good to me, thanks!